contract_recurring_invoicing_marker/__openerp__.py

```diff
@@ -6,7 +6,7 @@
     'name': 'Markers for contract recurring invoices',
     'version': '8.0.1.0.0',
     'category': 'Contract Management',
-    'author': 'Serv. Tecnol. Avanzados - Pedro M. Baeza, '
+    'author': 'Serv. Tecnol. Avanzados - Pedro M. Baeza, bisnesmart - Gonzalo Borrás,'
```

Comment:
Yeah, the exception was me, that my name was my company. But this is no more, as we have now Tecnativa, so the correct is: `Tecnativa, bisnesmart,`